components/Link.js

```diff
@@ -60,7 +65,7 @@ class Link extends React.Component {
             title={title}
             onClick={onClick}
             data-cy={this.props['data-cy']}
-            {...(!openInNewTab ? null : { target: '_blank', rel: 'noopener noreferrer' })}
+            {...(openInNewTab || this.state.isIframe ? { target: '_blank', rel: 'noopener noreferrer' } : null)}
```

Comment:
```suggestion
            {...(openInNewTab || this.isIframe() ? { target: '_blank', rel: 'noopener noreferrer' } : null)}
```

Reply:
`window` is only undefined in SSR, it's always defined in the browser. But if it doesn't work, I guess that means that the page is not re-rendering after re-hydratation. Let's go with your solution then!